tools/extra/pac/fpgabist/dma/fpga_dma_test.c

```diff
@@ -300,12 +476,10 @@ int main(int argc, char *argv[])
 
 out_destroy_tok:
    res = fpgaDestroyToken(&afc_token);
-   ON_ERR_GOTO(res, out_destroy_prop, "fpgaDestroyToken");
-
-out_destroy_prop:
-   res = fpgaDestroyProperties(&filter);
-   ON_ERR_GOTO(res, out, "fpgaDestroyProperties");
+   ON_ERR_GOTO(res, out, "fpgaDestroyToken");
 
+out_exit:
```

Comment:
see my prev. comment. You don't need to use retval. Use a consistent error type (fpga_result) throughout the application